fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnFileDecryptor.java

```diff
@@ -3,11 +3,12 @@
 import no.difi.meldingsutveksling.Decryptor;
 import no.difi.meldingsutveksling.IntegrasjonspunktNokkel;
 import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'no.difi.meldingsutveksling.config.IntegrasjonspunktProperties'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)